scripts/syntax.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
```

Comment:
Instead of writing this as a script, consider structuring the tests using Jasmine. 
The LSP end-to-end test is a good example of this.
This would allow us to achieve two things:
1. Use standardized testing procedures
2. Use TypeScript throughout the codebase

What do you think?

Reply:
Sounds good; I'll move it.